beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -44,37 +44,37 @@ npm start
 
 </TerminalBlock>
 
-For more information, [check out the official guide](https://create-react-app.dev/docs/getting-started).
+अधिक जानकारी के लिए, [आधिकारिक गाइड देखें](https://create-react-app.dev/docs/getting-started)।
 
-> Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline. This means you can use it with any backend you want. But if you're looking for more features like routing and server-side logic, read on!
+>  Create React App बनाएं बैकएंड लॉजिक या डेटाबेस को हैंडल नहीं करता है; यह सिर्फ एक फ्रंटएंड बिल्ड पाइपलाइन बनाता है। इसका मतलब है कि आप इसे अपने इच्छित किसी भी बैकएंड के साथ उपयोग कर सकते हैं। लेकिन अगर आप रूटिंग और सर्वर-साइड लॉजिक जैसी अधिक सुविधाओं की तलाश कर रहे हैं, तो पढ़ें!
```

Comment:
`Create React App` => `Create React App`

remove extra space at the start of the line